isb/isb_container_startup.sh

```diff
@@ -13,4 +13,4 @@ declare -p | grep -Ev 'BASHOPTS|BASH_VERSINFO|EUID|PPID|SHELLOPTS|UID' > /contai
 /usr/sbin/service cron start
 export PYTHONPATH=/app
 /usr/local/bin/python scripts/create_sitemaps_symlink.py -s /app/sitemaps  -d /app/isb_web/sitemaps
-uvicorn isb_web.main:app --host 0.0.0.0 --port 8000 --root-path /$ISB_UVICORN_ROOT_PATH --workers 12 --proxy-headers --forwarded-allow-ips '*'
\ No newline at end of file
+uvicorn isb_web.main:app --host 0.0.0.0 --port 8000 --root-path ''$ISB_UVICORN_ROOT_PATH --workers 12 --proxy-headers --forwarded-allow-ips '*'
```

Comment:
Do we need that empty string here?  That looks wrong to me.  Or is that just GitHub being weird?

Reply:
I added that because setting an empty string in the .env resulted in an unset environment variable. This seemed like a simple way to set the root path to an empty string if $ISB_UVICORN_ROOT_PATH was unset.

Reply:
Have you tested it when `$ISB_UVICORN_ROOT_PATH` *is* set?  I don't think those quotes will go away, will they?  So you'd end up with something like `''/smithsonian`?

Reply:
I think the shell script does some implicit string concatenation. I tested with the original UVICORN_ROOT_PATH=/sesar setting and all seemed to be working as it was before. If this isn't best practice I can find another solution

Reply:
Ok, if it works!

Reply:
This would seem to indicate is is behaving as you describe: https://stackoverflow.com/questions/4181703/how-to-concatenate-string-variables-in-bash

Reply:
Trying a simple test:

```
@app.get("/", include_in_schema=False)
async def root(request: fastapi.Request):
    return fastapi.responses.RedirectResponse(url=f"{request.scope.get('root_path')}/docs")
```

When it's `export ISB_UVICORN_ROOT_PATH=""`, we get `http://localhost:8000/docs`, which is good.

Reply:
And when it's `export ISB_UVICORN_ROOT_PATH="/smithsonian"`, we get `http://localhost:8000/smithsonian/docs`, which is also good.